src/storage/settings-save.ts

```diff
@@ -1,5 +1,9 @@
 /** Settings state for un/packing to/from LocalStorage. Not shareable. */
 export type SettingsSave = {
+  /** Which file system type to use when loading assets */
+  assetFilesystem: string
```

Comment:
I simplified this down to a string since this is the code that serializes to localStorage.  I'm fine with typing it, as it was before, but without the specific type in this one instance this file is pure with no imports.  I have no strong opinions either way, so I'll go with your preference.